.github/workflows/test.js.yml

```diff
@@ -55,14 +55,14 @@ jobs:
             NODE_AUTH_TOKEN: ${{ secrets.ACCESS_TOKEN }}
```

Comment:
Restore the original order we had.
First run migrations and then build the application

I don't think it matters in practice, but it makes more sense to be in this original order.